packages/js/plugins/http/src/schema.graphql

```diff
@@ -17,7 +17,7 @@ type Response {
 
 type Request {
   headers: [Header!]
-  urlParams: [UrlParam!]
```

Comment:
I think We can also do the same for header and replace it with Map

Reply:
Yup, gonna do that.